src/__tests__/01-connection.ts

```diff
@@ -44,13 +44,13 @@ it('I can connect to mongodb', async () => {
 it('I can connect to redis', async () => {
 	const value = "value"
 
-	redisClient.on("error", function(error) {
-		throw new Error("redis issue");
+	redis.on("error", function(error) {
+    console.log({error})
```

Comment:
we should probably use baseLogger here and add an error message for enhanced "queriness" if something goes wrong

Reply:
Is that needed for the test? It will automatically print it in the right place if the test fails, so we don't really need to search for anything here.

Reply:
oh yeah nevermind. I though the `redis.on("error", fn)` was on a non test file.

Reply:
that said, we probably still want to have the same thing in src/redis.ts to log redis error if they arise.